### PR TITLE
Homework 5

### PR DESCRIPTION
**Pull request by ilkivs**

### CODE REVIEW

File: src/HomeWork/hw5/Main.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package HomeWork.hw5;
+
+public class Main {
+    public static void main(String[] args) {
+        System.out.println("===== MyHashMap =====");
+        MyHashMap<String, Integer> myHashMap = new MyHashMap<>();
+        myHashMap.put("Sviatik", 21);
```
Comment (Siril-O) [inline src/HomeWork/hw5/Main.java:7]: Verify case when put entries with same key and case with entries with same hashcode

File: src/HomeWork/hw5/MyHashMap.java [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+    @Override
+    public V remove(K key) {
+        int hashCode = getHashCode(key);
+        int position = getPosition(hashCode);
+        if (nodes[position] == null) {
+            return null;
+        }
+        Node<K, V> currentNode = nodes[position];
+        Node<K, V> previousNode = currentNode;
```
Comment (Siril-O) [inline src/HomeWork/hw5/MyHashMap.java]: you can do this without previousNode

**Review by Siril-O — APPROVED** [review]

File: src/HomeWork/hw5/Main.java (hunk 1 above)
Comment (ist1997) [reply src/HomeWork/hw5/Main.java]: there is one case with entries with same key:
`        myHashMap.put("Sviatik", 21);
        myHashMap.put("Misha", 25);
        myHashMap.put("Vova", 25);

        System.out.println("put(Misha, 10000): ");
        myHashMap.put("Misha", 10000);
`

Comment (ilkivs) [reply src/HomeWork/hw5/Main.java:7]: there is one case with entries with same key
```
        myHashMap.put("Sviatik", 21);
        myHashMap.put("Misha", 25);
        myHashMap.put("Vova", 25);

        System.out.println("put(Misha, 10000): ");
        myHashMap.put("Misha", 10000);
```

Comment (ilkivs) [reply src/HomeWork/hw5/Main.java:7]: also i changed getHashCode() method, it returns always 2 and all the elements put in the same "bucket"

File: src/HomeWork/hw5/MyHashMap.java (hunk 2 above)
Comment (ilkivs) [reply src/HomeWork/hw5/MyHashMap.java]: changed

**Pull request merged**